### PR TITLE
Add option to skip python tests and execute bash tests instead

### PR DESCRIPTION
**Pull request by echen-98**
Description of changes: 

- Add option to skip python tests and execute bash tests instead: the default behavior remains to execute the python tests, but now that can be overridden if `SKIP_PYTHON_TESTS` is set `true`.
- Update usage message
- Remove invocation of python tests from `run-tests.sh` and clean up script

Regarding `run-tests.sh`: I removed the check for an `__init__.py` file since this check already exists in `kind-build-test.sh`; it was pretty confusing that even if `enable_python_tests` was set `false`, `run-tests.sh` would be executed and go on to execute python tests anyway (if they exist for the service). 

In other words, `enable_python_tests="true"` means "run the python tests inside Docker" – there's no change there. However, before, `enable_python_tests="false"` meant "run python tests normally if they exist; otherwise, run bash tests". With these changes, `enable_python_tests="false"` will actually mean "don't run python tests, but run bash tests instead". 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/kind-build-test.sh [hunk 1]
```diff
@@ -93,6 +94,11 @@ Environment variables:
                             Default: $ROOT_DIR/build/tmp-$CLUSTER_NAME
   K8S_VERSION               Kubernetes Version [1.14, 1.15, 1.16, 1.17, and 1.18]
                             Default: 1.16
+  TEST_HELM_CHARTS          Whether to run the test-helm.sh script (<true|false>)
```
Comment (jaypipes) [inline scripts/kind-build-test.sh:97]: :+1: thanks for adding this one.

File: test/e2e/run-tests.sh [hunk 2]
```diff
@@ -21,41 +27,21 @@ if [ $# -ne 1 ]; then
...
-  PYTEST_LOG_LEVEL="${PYTEST_LOG_LEVEL:-"INFO"}"
-  python bootstrap.py "${SERVICE}"
-
-  set +e
-
-  PYTHONPATH=. pytest -n auto --dist loadfile --log-cli-level "${PYTEST_LOG_LEVEL}" "${SERVICE}"
-  python cleanup.py "${SERVICE}"
-
-  set -eo pipefail
-fi
```
Comment (jaypipes) [inline test/e2e/run-tests.sh:61]: hmmm. by removing the above, you are removing the ability of the contributor to easily run the python tests on their local system instead of inside the Docker python testing image (which is how $TEST_E2E_DIR/build-run-test-dockerfile.sh works).

I suppose I don't mind too much being forced to use the Docker approach to running python tests, but wondering what 
@RedbackThomson thoughts are on this?

Comment (RedbackThomson) [reply test/e2e/run-tests.sh:61]: Yeah perhaps I don't understand why this block is being removed? It seems to just be removing the ability to run these tests locally?
I personally run tests locally because waiting for Docker and debugging inside a container is a pain.

Comment (echen-98) [reply test/e2e/run-tests.sh:61]: Yeah I thought about this a bit more, I agree keeping this would be useful. 

Since the bash tests are still useful to us (before we migrate everything to python), I found it inconvenient that `run-tests.sh` automatically ran python tests if they existed, even though the script was invoked because `enable_python_tests` was false in `kind-build-test.sh`.

I suppose there are currently 3 ways to run these tests (bash, python locally, python inside Docker), I'll try to come up with a way to allow the user to clearly specify one way.

**Comment by echen-98** [comment]
Ok hopefully this should be an improvement. The way to run tests is determined by `SKIP_PYTHON_TESTS` (default false, as before) and `RUN_PYTEST_LOCALLY` (default false but can change if needed), and the logic is abstracted to `run-tests.sh`.

File: scripts/kind-build-test.sh [hunk 3]
```diff
@@ -238,21 +248,7 @@ if [[ "$TEST_HELM_CHARTS" == true ]]; then
...
-if [[ "$enable_python_tests" == "false" ]]; then
-    # Run tests directly
-    $TEST_E2E_DIR/run-tests.sh $AWS_SERVICE
-else
-    # Run tests inside Dockerfile
-    $TEST_E2E_DIR/build-run-test-dockerfile.sh $AWS_SERVICE
-fi
+# run e2e tests
+export SKIP_PYTHON_TESTS="$SKIP_PYTHON_TESTS"
+export RUN_PYTEST_LOCALLY="$RUN_PYTEST_LOCALLY"
```
Comment (RedbackThomson) [inline scripts/kind-build-test.sh:253]: nit: These can be simplified down to just `export SKIP_PYTHON_TESTS` and `export RUN_PYTEST_LOCALLY`?

**Review by RedbackThomson — APPROVED** [review]
This is great! Thank you @echen-98 !

File: scripts/kind-build-test.sh (hunk 3 above)
Comment (echen-98) [reply scripts/kind-build-test.sh]: Done, thanks for the feedback!

**Review by jaypipes — APPROVED** [review]
Nice work, Eric. ++
